source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
 
-   For each message, the file name is generated based on the given
-   template. Then, this file is opened. As with the *file* property,
-   data is appended if the file already exists. If the file does not
-   exist, a new file is created. The template given in "templateName"
-   is just a regular :doc:`rsyslog template <../templates>`, so all
-   you have full control over how to format the file name.
-   
-   A cache of recent files is kept. Note
-   that this cache can consume quite some memory (especially if large
-   buffer sizes are used). Files are kept open as long as they stay
-   inside the cache.
-   Files are removed from the cache when a HUP signal is sent, the
-   *closeTimeout* occurs, or the cache runs out of space, in which case
-   the least recently used entry is evicted.
+========  ===============
+file:     "/path/to/file"
+Default:  none
+========  ===============
 
-.. function::  template [templateName]
+This parameters set in which file will Rsyslog write. It will always write into the same file.
 
-   *Default: template set via "template" module parameter*
+* If the file does not already exist, it is created.
+* If the file already exists, new data is appended to it.
 
-   Sets the template to be used for this action.
+**Important:** Files are kept open as long as rsyslogd is active.
+This conflicts with external log file rotation. In order to close a file after rotation, send rsyslogd a HUP signal after the file has been rotated away.
 
-.. function::  closeTimeout [minutes]
+=========  ===============
+dynaFile:  "template-name"
+Default:   none
+=========  ===============
 
-   *Default: for static files: 0; for dynamic files: 10*
+For each message, the file name is generated based on the given template, having full control over how to format filenames.
+**TODO Template must be a full path?**
    
-   *Available since: 8.3.3*
-
-   Specifies after how many minutes of inactivity a file is
-   automatically closed. Note that this functionality is implemented
-   based on the 
-   :doc:`janitor process <../../concepts/janitor>`.
-   See its doc to understand why and how janitor-based times are
-   approximate.
-
-.. function::  dynaFileCacheSize [size]
-
-   *Default: 10*
-
-   Applies only if dynamic filenames are used.
-   Specifies the number of DynaFiles that will be kept open.
-   Note that this is a per-action value, so if you have
-   multiple dynafile actions, each of them have their individual caches
-   (which means the numbers sum up). Ideally, the cache size exactly
-   matches the need. You can use :doc:`impstats <impstats>` to tune
-   this value. Note that a too-low cache size can be a very considerable
-   performance bottleneck.
-
-.. function::  zipLevel [level]
+A cache of recent files is kept. Note that this cache can consume quite some memory (especially if large buffer sizes are used). Files are kept open as long as they stay inside the cache.
```

Comment:
does the cache really eat up that much memory? this sounds like it was probably created a long time ago when system memory was much smaller. The filehandle is small, what else is there? 4K per open file?